src/frontend/src/routes/(app)/settings/relays/+page.svelte

```diff
@@ -1,62 +1,73 @@
 <script>
-	import { Plus, Server, XCircle } from "svelte-feathers";
-	import { slide } from "svelte/transition";
+  import { Server, XCircle } from "svelte-feathers";
+  import { slide } from "svelte/transition";
 
-	let relays = [
-		"wss://realay1.com",
-		"wss://realay2.com",
-		"wss://realay3.com",
-		"wss://realay4.com",
-		"wss://realay5.com",
-	];
+  //Load relays from store once ready
+  let relays = [
+    { url: "wss://realay1.com", status: "Connected" },
+    { url: "wss://realay2.com", status: "Connected" },
+    { url: "wss://realay3.com", status: "Loading" },
+    { url: "wss://realay4.com", status: "Failed" },
+    { url: "wss://realay5.com", status: "Connected" },
+  ];
 
-	let newRelay = "";
+  let newRelay = "";
 
-	// temporary delete and add functions
-	function deleteRelay(index) {
-		relays = [...relays.slice(0, index), ...relays.slice(index + 1)];
-	}
+  // temporary delete and add functions
```

Comment:
I am using VS code to format the code, previously I had bad setup (tabs or 4 spaces) now it is 2 spaces so the old code was with bad formatting. Hopefully it will be good from now on.